.gitignore

```diff
@@ -37,3 +37,4 @@ qgis-test.ctest
 qtcreator-build/
 scripts/Debug
 Thumb.db
+cmake-build-debug/*
```

Comment:
This is for my own builds, may I commit it or not?

Reply:
you can keep it in gitignore, np